sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs

```diff
@@ -110,6 +111,9 @@ private void Run()
         {
             try
             {
+                // Suppress the outbound Live Metrics service calls from being collected as dependency telemetry.
+                using var scope = SuppressInstrumentationScope.Begin();
```

Comment:
Should we move it here? - https://github.com/Azure/azure-sdk-for-net/blob/b4cec1cd78f765907fffd327f0ca2c39710b1871/sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.cs#L29